### PR TITLE
add: `freshfetch-git`

### PR DESCRIPTION
**Pull request by pythoncrazy**
I want to add freshfetch-git, for some reason it errors out on my ARM testing system, but works across the ~5 Ubuntu based distros running x86_64. If you have any advice/fixes, they would be very much appreciated.

### CODE REVIEW

**Comment by Elsie19** [comment]
What error is is on ARM?

**Comment by pythoncrazy** [comment]
Some error with the faccess crate, I am not sure how to fix it: 

```error[E0308]: mismatched types
    --> /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/faccess-0.2.3/src/lib.rs:95:36
     |
95   |             if faccessat(AT_FDCWD, path.as_ptr() as *const i8, mode, AT_EACCESS) == 0 {
     |                ---------           ^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `*const u8`, found `*const i8`
     |                |
     |                arguments to this function are incorrect
     |
     = note: expected raw pointer `*const u8`
                found raw pointer `*const i8`
note: function defined here
    --> /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/libc-0.2.104/src/unix/linux_like/linux/mod.rs:3738:12
     |
3738 |     pub fn faccessat(
     |            ^^^^^^^^^

For more information about this error, try `rustc --explain E0308`.
error: could not compile `faccess` (lib) due to previous error

File: packages/freshfetch-git/freshfetch-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+arch=('amd64')
+url="https://github.com/K4rakara/freshfetch"
+source=("https://github.com/K4rakara/freshfetch.git")
+maintainer=("pythoncrazy <pinakpaliwal07@gmail.com>")
+pkgver="0.2.0"
+makedepends=('cargo')
+external_connection="true"
+
+build() {
+  cd "${_archive}"
```
Comment (Elsie19) [inline packages/freshfetch-git/freshfetch-git.pacscript:13]: Stop using _archive, use `"${gives}"` or `"freshfetch"`.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/freshfetch-git/freshfetch-git.pacscript (hunk 1 above)
Comment (pythoncrazy) [reply packages/freshfetch-git/freshfetch-git.pacscript:13]: Alright, will do

File: packages/freshfetch-git/freshfetch-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,20 @@
+pkgname="freshfetch-git"
+gives="freshfetch"
+pkgdesc="A fresh take on Neofetch"
+arch=('amd64')
+url="https://github.com/K4rakara/freshfetch"
```
Comment (Elsie19) [inline packages/freshfetch-git/freshfetch-git.pacscript:5]:
```suggestion
url='https://github.com/K4rakara/freshfetch'
```

File: packages/freshfetch-git/freshfetch-git.pacscript [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+source=("https://github.com/K4rakara/freshfetch.git")
+maintainer=("pythoncrazy <pinakpaliwal07@gmail.com>")
+pkgver="0.2.0"
+makedepends=('cargo')
+external_connection="true"
+
+build() {
+  cd "${gives}"
+  cargo build --release -vv
```
Comment (Elsie19) [inline packages/freshfetch-git/freshfetch-git.pacscript:14]:
```suggestion
  cargo build --locked --release -vv
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**